packages/shared/src/modules/categories/libs/types/category-update-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type CategoryCreateRequestDto } from "./category-create-request-dto.type.js";
+
+type CategoryUpdateRequestDto = Partial<CategoryCreateRequestDto>;
```

Comment:
Why not just use Partial<CatergoryCreateReqestDto> where this one is used?

Reply:
This is to make it more convenient to change the type of the update request if needed.